ga4gh/client.py

```diff
@@ -46,17 +61,14 @@ def _usingWorkaroundsFor(self, workaround):
         return workaround in self._workarounds
 
     def _debugResponse(self, jsonString):
-        if self._debugLevel > 1:
-            # TODO use a logging output and integrate with HTTP client more
-            # nicely... also, logging / error handling in general
-            print("json response:")
-            pp = json.dumps(
-                json.loads(jsonString), sort_keys=True, indent=4)
-            print(pp)
+        self.logger.debug("json response:")
+        pp = json.dumps(
```

Comment:
Hm, you're right.  I would have liked to just send everything to the logger and let its internal configuration determine what and what not to emit, but it looks like we will need to add in some of our own logic for performance reasons.

@jeromekelleher do you think it makes more sense to drive the logic off of `_debugLevel` or off of logger's log levels e.g. `if self.logLevel < logger.WARN` ?  If we do the former than _debugLevel only is used in __init__ and nowhere else, which is nice... on the other hand, doing the later forces us to understand more about how logger works...